<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>451</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyWidgetRegister" name="LoginInner" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>662</width>
     <height>442</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>662</width>
     <height>442</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>662</width>
     <height>442</height>
    </size>
   </property>
   <widget class="QWidget" name="LoginContent" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>662</width>
      <height>367</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>662</width>
      <height>367</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>662</width>
      <height>367</height>
     </size>
    </property>
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>281</width>
       <height>361</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>扫一扫登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_double_code">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>154</width>
        <height>154</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="imagelogin.qrc">:/Login/C:/Users/ASUS/Pictures/Login/code.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>请使用</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>扫码登录</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>88</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>简·活动App</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidgetLeft">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageId">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>账号密码登录</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButtonTonote">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>141</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>短信快捷登录</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>自动登录</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>210</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>忘记密码？</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_login_id_and_pwd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>291</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QWidget" name="widgetId" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_id">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imagelogin.qrc">:/Login/C:/Users/ASUS/Pictures/Login/human-1-fill-gray.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyComboBox" name="comboBox_id">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>zhangsan</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetPwd" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_pwd">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="imagelogin.qrc">:/Login/C:/Users/ASUS/Pictures/Login/lock.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyLineEdit" name="lineEdit_password">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>123321</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="pageShortMsg">
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>短信快捷登录</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButtonToId">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>账号密码登录</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>290</width>
         <height>40</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>5</y>
          <width>241</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>290</width>
         <height>40</height>
        </rect>
       </property>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>42</x>
          <y>5</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_11">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>5</y>
          <width>121</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>获取动态密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>290</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QToolButton" name="toolButtonRegister">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>320</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imagelogin.qrc">
       <normaloff>:/Login/C:/Users/ASUS/Pictures/Login/ufufill.png</normaloff>:/Login/C:/Users/ASUS/Pictures/Login/ufufill.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_head">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>简·活动</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imagelogin.qrc">
      <normaloff>:/Login/C:/Users/ASUS/Pictures/Login/wkf.png</normaloff>:/Login/C:/Users/ASUS/Pictures/Login/wkf.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>--</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imagelogin.qrc">
      <normaloff>:/Login/C:/Users/ASUS/Pictures/Login/set2.png</normaloff>:/Login/C:/Users/ASUS/Pictures/Login/set2.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>logo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyWidgetRegister</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetregister.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagelogin.qrc"/>
 </resources>
 <connections/>
</ui>
